webrtc/src/track/track_local/track_local_static_rtp.rs

```diff
@@ -11,6 +11,7 @@ pub struct TrackLocalStaticRTP {
     codec: RTCRtpCodecCapability,
     id: String,
     stream_id: String,
+    paused: Mutex<Option<Arc<AtomicBool>>>,
```

Comment:
This should live inside `TrackBinding` I think. One `TrackLocalStaticRTP` can be bound to several senders(which are the things actually being paused), hence you can have a case where one sender is paused, but the other isn't it

Reply:
Can it?

It seems to me the multiple `TrackBinding` is because we can send on multiple PT (codecs?) at the same time. All of them share the same `SSRC` though, which means we are still talking about the same m-line.

And the `paused` is derived by direction, which is m-line… so it seems right. Or?

Reply:
Or are you saying there can be multiple call to `bind` for the same `TrackLocal`?

Reply:
I think so, but let me double check that my understanding is correct

Reply:
Ok. I can see that. In that case we only need the pause in the TrackBinding, since the `TrackLocalSample` has an internal such anyway.